.changeset/little-rabbits-wave.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/cli': patch
+---
+
+Add --format option to backstage-cli config:schema command.
```

Comment:
No need to changeset this actually, since we already have a pending one that introduces the change as intended

Reply:
dropped.